app/presenters/notifications/setup/scheduled-notifications.presenter.js

```diff
@@ -0,0 +1,140 @@
+'use strict'
+
+/**
+ * Formats a notification into a 'water.scheduled_notifications'
+ * @module ScheduledNotificationPresenter
+ */
+
+const { notifyTemplates } = require('../../../lib/notify-templates.lib.js')
+const { timestampForPostgres } = require('../../../lib/general.lib.js')
+
+const INVITATIONS_MESSAGE_REF = {
+  [notifyTemplates.returns.invitations.primaryUserEmail]: 'returns_invitation_primary_user_email',
+  [notifyTemplates.returns.invitations.returnsAgentEmail]: 'returns_invitation_returns_agent_email',
+  [notifyTemplates.returns.invitations.licenceHolderLetter]: 'returns_invitation_licence_holder_letter',
+  [notifyTemplates.returns.invitations.returnsToLetter]: 'returns_invitation_returns_to_letter'
+}
+
+//   returnReminder: {
+//     [CONTACT_ROLE_PRIMARY_USER]: emailTemplate('returns_reminder_primary_user_email'),
+//     [CONTACT_ROLE_RETURNS_AGENT]: emailTemplate('returns_reminder_returns_agent_email'),
+//     [CONTACT_ROLE_LICENCE_HOLDER]: letterTemplate('returns_reminder_licence_holder_letter'),
+//     [CONTACT_ROLE_RETURNS_TO]: letterTemplate('returns_reminder_returns_to_letter')
+//   }
+// }
+
+/**
+ * Formats a notification into a 'water.scheduled_notifications'
+ *
+ * @param {object} notification
+ * @param {string} journey
+ *
+ * @param notify
+ * @returns {object}
+ */
+function go(notification, journey, notify) {
+  return {
+    messageType: 'letter',
+    messageRef: _messageRef(journey, notification.templateId),
```

Comment:
Need to add / check if we need 

- status_checks
- next_status_check
- notification_type
- job_id